packages/desktop-gui/cypress-component.json

```diff
@@ -0,0 +1,14 @@
+{
+  "viewportWidth": 800,
+  "viewportHeight": 850,
+  "supportFile": "../../node_modules/cypress-react-unit-test/support",
+  "pluginsFile": "../../node_modules/cypress-react-unit-test/plugins/load-webpack",
+  "env": {
+    "webpackFilename": "webpack.config.ts",
+    "coverage": false
+  },
+  "testFiles": "**/*-spec.{js,jsx}",
+  "experimentalComponentTesting": true,
+  "componentFolder": "src",
+  "nodeVersion": "system"
+}
```

Comment:
it doesn't feel right that we need to create a new `cypress-component.json` file. ideally this should be baked into the existing `cypress.json`. Our desktop gui tests are pretty vanilla.